Makefile

```diff
@@ -666,5 +653,6 @@ fetch-sample-data:
 	unzip -o sample-data-1-0.zip
 	rm -f sample-data-1.0.zip || true
 
-generate-sample-data:
+generate-sample-data: build-netstandard
 	@echo Generate Sample Data
+	dotnet test -a $(PROCESSOR) -c $(TARGET) --filter TestCategory=OutputTestData ./bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/ElectionGuard.Decryption.Tests.csproj
```

Comment:
this will run an end to end and dump all the data out to `bin/TARGET/data/generated`. we should copy it to the data folder and remove all the files that i committed to source as part of this pr (and the previous one) i jsut havent had a chancve to do that yet since it also affects the typescript tests